src/GraphQLSchemaLinter.php

```diff
@@ -79,7 +79,12 @@ public function getVersion() {
     if (!empty($this->dependencyVersions)) {
 
       foreach ($this->dependencyVersions as $name => $required) {
-        list($err, $dep_version, $stderr) = exec_manual('cd .arcanist 2>&1 >/dev/null && npm list --depth=0 2>/dev/null | grep "%C" | cut -d "@" -f 3 | tr -d "[:space:]"', $name);
+        $bin_root = dirname($this->getDefaultBinary());
+        list($stdout) = new ExecFuture('npm list -s --depth=0 --json || exit 0')
```

Comment:
To use the fluid style, this should be:

```php
list($stdout) = newv('ExecFuture', $args)
```

If you want to use `new ExecFuture` directly, the you'll need to use an intermediate `$future` value.

(My mistake in the previous suggestion.)